app/views/sites/usage_rules/edit.html.slim

```diff
@@ -1,60 +1,81 @@
-- content_for(:title)
-  = "Usage Rules"
-
+- content_for :title, "Usage Rules"
 - content_for :page_header_title, "Usage Rules"
 
-#usage-rules
-  = semantic_form_for @settings, url: admin_site_usage_rules_path do |form|
-
-    = form.inputs 'User Account Management Zone' do
-      = form.input :useraccountarea_enabled
-
-    - if current_account.master? && !ThreeScale.config.onpremises
-      = form.inputs 'Hide services' do
-        = form.input :hide_service
-
-    = form.inputs 'Signup' do
-      = form.input :signups_enabled
-      - if @settings.authentication_strategy.cas?
-        = form.input :cas_server_url
-
-      - sso_hint = current_account.authentication_providers.any? ? ' ' + t('.sso_integrations_info_html', link: provider_admin_authentication_providers_path) : ''
-      - if @settings.approval_required_editable?
-        = form.input :account_approval_required,
-                     as: :boolean,
-                     hint: t("formtastic.hints.settings.account_approval_required") + sso_hint
-      - elsif @settings.approval_required_disabled?
-        = form.input :account_approval_required,
-                     as: :boolean,
-                     :input_html => { :disabled => true },
-                     hint: t("formtastic.hints.settings.approval_required_referer",
-                             link: link_to("Account Plans", admin_buyers_account_plans_path) ) + sso_hint
-
-    = form.inputs 'Users' do
-      = form.input :strong_passwords_enabled, hint: t("formtastic.hints.settings.strong_passwords_enabled", strong_password_definition: strong_password_definition)
-
-    = form.inputs "Search" do
-      = form.input :public_search
+div class="pf-c-card"
+  div class="pf-c-card__body"
+    = semantic_form_for @settings, url: admin_site_usage_rules_path, html: { class: "pf-c-form pf-m-limit-width" } do |form|
+      section class="pf-c-form__section" role="group" aria-labelledby="settings-section-user-account"
+        div class="pf-c-form__section-title" id="settings-section-user-account" aria-hidden="true" User account management zone
```

Comment:
I think `h2` would look nicer here (well, for all sections), and semantically better?...

Reply:
I see what you mean but Patternfly disagrees: https://www.patternfly.org/v4/components/form/html#form-sections 🤷🏻

Reply:
yeah, but https://www.patternfly.org/v4/components/form/html/#usage says that `.pf-c-form__section-title` is applicable to multiple elements, `h2` included :innocent: 

also: https://github.com/patternfly/patternfly/issues/3972

So, I think it's at our discretion...

Reply:
Fair enough, let's use `h2` then. And also `label` instead of `div`.